README.md

```diff
@@ -6,151 +6,107 @@ This repository holds the kcp tests that tests against the publicly available  i
-default                                            Active   4h46m
-e2e-test-olm-a-a92jyymd-lmgj6                      Active   4m28s
-e2e-test-olm-a-a92jyymd-pr8hx                      Active   4m29s
-...
-```
-#### Print cluster event on Terminal
-When you execute cases, there are some cluster event which is printed to the terminal, like
+### Create go-bindata for new YAML files
+If you have some **new YAML files** used in your code, you have to generate the bindata first.
+Run `make update` to update the bindata. For example, you can see the bindata has been updated after running the `make update` as follows:
 ```console
-Timeline:
-
-Mar 30 03:57:36.435 I ns/openshift-kube-controller-manager pod/kube-controller-manager-ip-10-0-190-60.ec2.internal created SCC ranges for e2e-test-olm-common-l21c9cfo-g6xwx namespace
-Mar 30 03:57:47.894 W ns/openshift-marketplace pod/marketplace-operator-5cf7b79dd4-xsffg node/ip-10-0-247-215.ec2.internal graceful deletion within 30s
-Mar 30 03:57:48.097 I ns/openshift-marketplace pod/marketplace-operator-5cf7b79dd4-xsffg Stopping container marketplace-operator
-...
+$ git status
+	modified:   test/extended/testdata/bindata.go
+	new file:   test/extended/testdata/kcp/xxxx.yaml
 ```
-Someone does not want it on the terminal, but someone wants it for debugging.
-
-So, we add environment variable ENABLE_PRINT_EVENT_STDOUT to enable it.
-
-In default, it does not print the cluster event on the terminal when you execute the case on your terminal.
 
 ### Compile the executable binary
 Note that we use the `go module` for package management, the previous `go path` is deprecated.
 ```console
-$ git clone git@github.com:openshift/openshift-tests-private.git
-$ cd openshift-tests-private/
+$ git clone git@github.com:kcp-dev/kcp-tests.git
+$ cd kcp-tests/
 $ make build
 mkdir -p "bin"
-export GO111MODULE="on" && export GOFLAGS="" && go build -o "bin" "./cmd/extended-platform-tests"
-$ ls -hl ./bin/extended-platform-tests 
--rwxrwxr-x. 1 cloud-user cloud-user 165M Jun 24 22:17 ./bin/extended-platform-tests
+export GO111MODULE="on" && export GOFLAGS="" && go build  -ldflags="-s -w" -mod=mod -o "bin" "./cmd/kcp-tests"
+$ ls -hl ./bin/kcp-tests 
+-rwxrwxr-x. 1 cloud-user cloud-user 106M Sep 13 14:41 ./bin/kcp-tests
 ```
 
 ### Run the automation test case
-The binary finds the test case via searching for the test case title. It searches the test case titles by RE (`Regular Expression`). So, you can filter your test cases by using `grep`. Such as, if I want to run all [OLM test cases](https://github.com/openshift/openshift-tests-private/blob/master/test/extended/operators/olm.go#L21), and all of them contain the `OLM` letter, I can use the `grep OLM` to filter them, as follows: 
+The binary finds the test case via searching for the test case title. It searches the test case titles by RE (`Regular Expression`). So, you can filter your test cases by using `grep`. 
+##### Run an area automation test cases
```

Comment:
@Phaow may be change this to Run automation cases related to an area ?

Reply:
Addressed.